src/js/view/controls/controls.js

```diff
@@ -71,10 +70,20 @@ export default class Controls {
                 this.activeTimeout = setTimeout(this.userInactiveTimeout, remainingTime);
                 return;
             }
+            if (this.playerContainer.querySelector('.jw-tab-focus')) {
```

Comment:
@DanFerrer why do we need this ? We already set the activeTimeout to 0 when tabbing inside the player 
https://github.com/jwplayer/jwplayer/blob/8bef82ffd5e0dbb64c61906476bbe3df2e6b3114/src/js/view/controls/controls.js#L325